tests/test_app.py

```diff
@@ -27,8 +33,12 @@ class TestHitNext:
         """Provide 'rh_account' in metadata."""
         resp = await app.hit_next(
             'STUB_ID',
-            {'url': 'http://STUB.URL', 'rh_account': 'STUB_ACCOUNT'}
-        )
+            {
+                'url': 'http://STUB.URL',
+                'rh_account': 'STUB_ACCOUNT',
+                'b64_identity': 'abcde'
+            }
+            )
         request = stub_server['requests'][0]
 
         assert request['content']['metadata']['rh_account'] == 'STUB_ACCOUNT'
```

Comment:
Missing assert on headers. It may need to change the test server fixture to remember headers as well.